### DISCUSSION TITLE
Loving the app so far

### DISCUSSION THREAD

**Discussion by danz0l99** [docmost/docmost#82] [General]
Wow, I absolutely love what you have done here.  Basically Docusaurus without dealing with plan markdown with some added advantages. 

It would be amazing for public-facing websites when it's matured a little more to create public HTML (its so close already). Until then I'll use it as a way of creating an easy markdown files that I can use elsewhere and continue to use it as a local wiki.

Excellent job

**Comment by Philipinho** [comment]
Thank you for your feedback.
I have plans for the public-facing aspect of it.

**Reply by AMDphreak** [reply]
good to hear. This guy was asking for it too. https://github.com/docmost/docmost/discussions/123#discussion-6976455

**Reply by Nasjoe** [reply]
Same here : https://github.com/docmost/docmost/discussions/608
Can't wait to test this :)
Thanks !
